ingestion/src/metadata/ingestion/source/dashboard/redash/metadata.py

```diff
@@ -235,11 +235,17 @@ def yield_dashboard_lineage_details(
                     for table in lineage_parser.source_tables:
                         table_name = str(table)
                         database_schema_table = fqn.split_table_name(table_name)
+                        database_schema_name = (
+                            None
+                            if database_schema_table.get("database_schema")
+                            == "<default>"
+                            else database_schema_name
+                        )
```

Comment:
since we are using this logic at multiple place it would make sense to create a util function out of this and use that everywhere this was we can also add a unit test for it

Reply:
Done